src/classes/RLLP_OppAccRollup_BATCH.cls

```diff
@@ -36,8 +36,14 @@ public with sharing class RLLP_OppAccRollup_BATCH implements Database.Batchable<
 		
 	public Boolean executeReturnStatus() {
         if(UTIL_JobScheduler.canRunBatch()) {
-            Database.executeBatch(this);
-            return true;
+        	Savepoint sp = Database.setSavepoint();
+        	try {
+                Database.executeBatch(this);
+                return true;
+        	} catch(Exception e) {
+	            Database.rollback(sp);
+	            ERR_Handler.processError(e, ERR_Handler_API.Context.RLLP);
```

Comment:
I created another issue to track the work on getting the batch jobs to record errors and send notifications: https://github.com/SalesforceFoundation/Cumulus/issues/1294

It'll most likely require to rewrite the widget.